lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_Standard_PMPSDB.TcPOU

```diff
@@ -64,6 +66,12 @@ MOTION_GVL.fbPmpsFileReader(
 );
 
 ftBusy(CLK:=MOTION_GVL.fbPmpsFileReader.bBusy);
+IF ftBusy.Q THEN
+    bReadPmpsDb := TRUE;
+ELSE
+    bReadPmpsDb := FALSE;
+END_IF
```

Comment:
Uhh I don't want to take back my merge, but next time note that you can do this in one line:
```
bReadPmpsDb := ftBusy.Q
```

Reply:
Ah yes, that is cleaner. Thanks for pointing that out. Will use this next time.